target_node.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# If the script is not run with sudo, stop & warn
+if [ "$EUID" != 0 ]; then
+    echo "This script must be run as root"
+    exit 1
+fi
+
+# Install ssh server
+apt install -y openssh-server
+
+# Ensure ssh is running
+systemctl status ssh --no-pager
```

Comment:
Ensure ssh is really reboot safe ;)
```suggestion
systemctl status ssh --no-pager
systemctl enable ssh
```